numpy/core/src/private/ufunc_override.c

```diff
@@ -134,6 +134,7 @@ PyUFunc_WithOverride(PyObject *args, PyObject *kwds,
                 goto fail;
             }
             if (with_override != NULL) {
+                Py_INCREF(obj);
```

Comment:
Seems this is only needed if obj can be destroyed before used. Is that the case?